css/style.css

```diff
@@ -1,8 +1,79 @@
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- */
+
+body{
+font-family: 'roboto', Times, serif
+}
+/* ** Header Start ** */
+.title{
+color: rgb(176, 19, 1);
+font-size: 30px;
+}
+.desc{
+font-size: 17px;  
+}
+.navbar{
+margin-left: 50px;
+margin-right: 50px;
+padding-left: 15px;
+padding-right: 15px;
+max-width: max-content;
+background-color: #EFEFEF;
+border-radius: 10px;
+display: flex;
+flex-wrap: wrap;
+flex-direction:column;
+}
+.navigation:hover{
+    color:rgb(176, 19, 1);
+}
+
+/* ** Header End ** */
+/* ** Article Start ** */
+.article {
+padding-left: 50px;
+padding-right: 50px;
+margin-top: 30px;    
+}
+.articleImg{
+max-width: 100%;
+width: 700px;
+height: 400px;
+max-height: 100%;
+border-radius: 15px;   
+object-fit: cover;
+}
+h2:first-child{
+    border-bottom: 5px solid rgb(176, 19, 1);
+    width: 50%;
+}
+.articletext{
+max-width: 100%;
+width: 700px;
+}
+h5{
+color: rgba(107, 107, 107, 0.794);
+}
+.articleBtn{
+border: none;
+border-radius: 5px;
+background-color: rgb(176, 19, 1);
+color: white;
+font-weight: 500;
+padding: 10px;
+}
+.articleBtn:hover{
+    background-color: white;
+    color:rgb(176, 19, 1);
+    border: 2px solid rgb(176, 19, 1);
```

Comment:
Try moving this into the .articleBtn selector and see what happens. It might behave more 'nicely' in terms of the box not jiggling around on the screen as it transitions between having no border and having a 2px border